bundle-workflow/tests/tests_system/test_arch.py

```diff
@@ -15,6 +15,18 @@ class TestArch(unittest.TestCase):
     def test_current_arch(self):
         self.assertTrue(current_arch() in ["x64", "arm64"])
 
+    @patch("subprocess.check_output")
```

Comment:
I think you can do `@patch("subprocess.check_output", return_value=...)` which would make things a bit cleaner.

Reply:
I will address this in another improvement PR.